labgrid/pytestplugin/fixtures.py

```diff
@@ -52,6 +54,37 @@ def env(request):
     env = Environment(config_file=lg_env)
     if lg_coordinator is not None:
         env.config.set_option('crossbar_url', lg_coordinator)
+
+    if pytest.config.pluginmanager.hasplugin('junitxml'):
+        my_junit = getattr(pytest.config, '_xml', None)
+
+        my_junit.add_global_property('YAML_CONFIG', env.config_file)
+        for i,j in enumerate(env.config.get_targets()):
+            try:
+                name = env.config.get_targets()[j]['resources']['RemotePlace']['name']
+                my_junit.add_global_property('TARGET_{}_REMOTE'.format(i), name)
+            except KeyError:
+                pass
+            my_junit.add_global_property('TARGET_{}'.format(i), j)
+
+        for name,path in env.config.get_paths().items():
+            try:
+                sha = subprocess.check_output("git show --format=%H -s".split(" "),cwd=path)
+            except subprocess.CalledProcessError:
+                continue
+            except FileNotFoundError:
+                continue
+            my_junit.add_global_property('PATH_{}'.format(name.upper()),sha.decode("utf-8").strip("\n"))
```

Comment:
We should always export the path itself. The commit could be exported under PATH_name_GIT_COMMIT.